marketplace-kit-data-export.js

```diff
@@ -9,21 +9,13 @@ const program = require('commander'),
   version = require('./package.json').version;
 
 let gateway;
-const spinner = ora({
-  text: 'Exporting',
-  stream: process.stdout,
-  spinner: 'bouncingBar'
-});
+const spinner = ora({ text: 'Exporting', stream: process.stdout, spinner: 'bouncingBar' });
 const transform = data => {
-  const results = {
-    users: [],
-    transactables: [],
-    models: []
+  return {
```

Comment:
```const transform = data => ({ users = { results: [] }, transactables = { results: [] }, models = { results: [] } }) => {
return {
  users: users.results,
  transactables: transactables.results,
  models: models.results 
})```

if data.users / data.transactables / data.models keys are missing it will blow up, thats why defaults are good idea

Reply:
@pavelloz Fixed.